### PR TITLE
CDCG-28 - add-Pagination-ModulePads

### PR DESCRIPTION
**Pull request by IsaacGalindoBarbina**
### Feature / Bug Description
No habia una paginacion para todos los pads, por lo que se realizo una para que fuera mas sencillo entonces un pad.

### Solution
Creacion de una paginacion y cada paginacion traen 20 registros cada pagina. 

### Notes
Add some notes

### Tickets
[CDCG-28](https://cdentalcaregroup.atlassian.net/browse/CDCG-28)

### Screenshots / Screencasts
CDCG-28-ModulePads

[CDCG-28]: https://cdentalcaregroup.atlassian.net/browse/CDCG-28?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/pages/pad/Pads.tsx [hunk 1]
```diff
@@ -1,266 +1,338 @@
...
+
+	const handlePageChange = (page: number) => {
+		setCurrentPage(page);
+	};
+
+	const handleGetPads = async () => {
+		try {
+			const response = await getPads({}).unwrap();
+			if (props.rol == UserRoles.RECEPTIONIST) {
+				console.log(`Process rep pads`);
```
Comment (imadiaz) [inline src/pages/pad/Pads.tsx:88]: can you remove this log?

File: src/pages/pad/Pads.tsx [hunk 2]
```diff
@@ -1,266 +1,338 @@
...
+				setAditionalMembers(dataList);
+				setIsListLoading(false);
+			}, 100);
+		} else {
+			handleErrorNotification(Constants.MAX_MEMBERS_PAD);
+		}
+	};
+
+	const handleOnAddAditionalMember = async () => {
+		//console.log(aditionalMembers)
```
Comment (imadiaz) [inline src/pages/pad/Pads.tsx:203]: also these one please

File: src/pages/pad/Pads.tsx (hunk 1 above)
Comment (IsaacGalindoBarbina) [reply src/pages/pad/Pads.tsx]: Okey, don't worry.
I have already deleted the log

Comment (IsaacGalindoBarbina) [reply src/pages/pad/Pads.tsx]: Okey, don't worry,

**Comment by IsaacGalindoBarbina** [comment]
El video se muestra la paginacion de las cards de los pads
[Uploading CDCG-28-UpdatePagination.zip…]()
@imadiaz @immanueldiaz

**Comment by imadiaz** [comment]
> El video se muestra la paginacion de las cards de los pads Uploading CDCG-28-UpdatePagination.zip… @imadiaz @immanueldiaz https://www.veed.io/view/33a01aad-c073-4964-bf8c-39f5d1135d2a?panel=share

Thank u! according with the video its okey but after the user change the pagination we need to scroll up to the page, in the video you keep the scroll in the bottom of the page, after to change in any page we need to redirect at the top, you can use the windows functions default in JS

**Comment by IsaacGalindoBarbina** [comment]
Well, I think I already have the module on scrolling and pagination from the Pads module and I'm attaching a video.
https://www.veed.io/view/9a8c243e-59d1-404c-98f7-48e8dd513406?panel=share

**Review by imadiaz — APPROVED** [review]

**Review by immanueldiaz — APPROVED** [review]

**Review by ThayliAreliVillegas112 — APPROVED** [review]
Nice!

**Pull request merged**